src/components/RankingPrompt/RankingPrompt.js

```diff
@@ -6,6 +6,15 @@ import { addUserToRanking } from '../../shared/ranking';
 import styles from './RankingPrompt.module.css';
 import charactersCelebrationUrl from '../../public/img/CharactersCelebration.png';
 
+const id = Date.now();
```

Comment:
Jest bug i każdy gracz, w trakcie jednej sesji będzie miał teraz to samo id (bo jest tworzone jedno globalne). Najprostszy sposób reprodukcji: rozpocząć quiz -> zamknąć -> zapisać wynik jako "gracz1" -> kliknąć "spróbuj ponownie" -> zamknąć -> zapisać wynik jako "gracz2" -> kliknąć "ranking" -> obaj gracze mają to samo id.

Musiałbyś np. tworzyć id tam gdzie było poprzednio, przyjąć je w `closePopup` w `Answers.js` (już jest tam podawane przez `close(id)` jako argument), zapisać w `Answers` w zmiennej lokalnej w komponencie i tą zmienną lokalną podać do routera.

Reply:
nie wiem jak tego nie zauważyłem. Zmieniłem, chyba lepiej niż teraz już nie wymyślę.